### PR TITLE
Redesign students details and edit components

### PR DESCRIPTION
**Pull request by VladyslavPolushyn**
# Summary

- changed color of buttons
- changed color of closed icon in input

### CODE REVIEW

File: src/features/students/edit-students-details/edit-students-details.js [hunk 1]
```diff
@@ -258,13 +258,13 @@ export const EditStudentsDetails = ({ id }) => {
                             {groups.map(({ id, name }) => (
                               <li
                                 className={classNames(styles['list-element'],
-                                  'd-flex bg-light border border-outline-secondary rounded')}
+                                  'd-flex bg-light border border-outline-secondary rounded font-weight-bold')}
```
Comment (Icemandov) [inline src/features/students/edit-students-details/edit-students-details.js:261]: It looks better without `font-weight-bold`

File: src/features/students/edit-students-details/edit-students-details.js [hunk 2]
```diff
@@ -258,13 +258,13 @@ export const EditStudentsDetails = ({ id }) => {
...
                                 key={id}
                                 data-groupid={id}
                                 data-groupname={name}
                               >{name}
                                 <button
-                                  className="btn p-0 ml-auto mr-2 font-weight-bold text-danger"
+                                  className="btn p-0 ml-auto mr-2 font-weight-bold text-muted"
```
Comment (Icemandov) [inline src/features/students/edit-students-details/edit-students-details.js:267]: Better use `text-dark`

File: src/features/students/edit-students-details/edit-students-details.js [hunk 3]
```diff
@@ -279,7 +279,7 @@ export const EditStudentsDetails = ({ id }) => {
                       <div className="col-md-3 col-4">
                         <Button
                           className="w-100"
-                          variant="danger"
+                          variant="dark"
```
Comment (PavelKundenko) [inline src/features/students/edit-students-details/edit-students-details.js:282]: Better use `$theme-colorDark`

File: src/features/students/edit-students-details/edit-students-details.js (hunk 1 above)
Comment (VladyslavPolushyn) [reply src/features/students/edit-students-details/edit-students-details.js]: Thank's, done!

File: src/features/students/edit-students-details/edit-students-details.js (hunk 2 above)
Comment (VladyslavPolushyn) [reply src/features/students/edit-students-details/edit-students-details.js]: Done

File: src/features/students/edit-students-details/edit-students-details.js (hunk 3 above)
Comment (VladyslavPolushyn) [reply src/features/students/edit-students-details/edit-students-details.js]: Thank's!
Done.

**Review by R0D1N — APPROVED** [review]

**Review by MarinaPasternak — APPROVED** [review]

**Review by CoderGTR — APPROVED** [review]

**Review by PavelKundenko — APPROVED** [review]

**Review by Icemandov — APPROVED** [review]

**Pull request merged**